### PR TITLE
evm: Track number of attestations

### PR DESCRIPTION
**Pull request by bruce-riley**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -279,9 +279,9 @@ The Endpoint MUST contain the following functionality for an Integrator:
   - MUST have at least one enabled **send** Adapter for `dstChain`.
   - Increments the Integrator's sequence and performs the steps to send the message or prepare it for sending, as applicable.
   - If Adapters must pull outgoing messages in the given implementation (via `pickUpMessage`), the Endpoint MUST set the current enabled Send Adapters as the Outstanding Adapters for that message.
-- `getMessageStatus(srcChain, srcAddr, sequence, dstChain, dstAddr, payloadHash)` → `enabledBitmap, attestedBitmap, executed`
+- `getMessageStatus(srcChain, srcAddr, sequence, dstChain, dstAddr, payloadHash)` → `enabledBitmap, attestedBitmap, numAttested, executed`
   - Returns the enabled receive Adapters for that chain along with the attestations and the executed flag.
```
Comment (panoel) [inline README.md:283]:
```suggestion
  - Returns the enabled receive Adapters for that chain along with the attestation information and the executed flag.
```

File: README.md [hunk 2]
```diff
@@ -279,9 +279,9 @@ The Endpoint MUST contain the following functionality for an Integrator:
   - MUST have at least one enabled **send** Adapter for `dstChain`.
   - Increments the Integrator's sequence and performs the steps to send the message or prepare it for sending, as applicable.
   - If Adapters must pull outgoing messages in the given implementation (via `pickUpMessage`), the Endpoint MUST set the current enabled Send Adapters as the Outstanding Adapters for that message.
-- `getMessageStatus(srcChain, srcAddr, sequence, dstChain, dstAddr, payloadHash)` → `enabledBitmap, attestedBitmap, executed`
+- `getMessageStatus(srcChain, srcAddr, sequence, dstChain, dstAddr, payloadHash)` → `enabledBitmap, attestedBitmap, numAttested, executed`
```
Comment (panoel) [inline README.md:282]: Question:  So, the `attestedBitmap` is a map of the receive transceivers that have attested this message and the `numAttested` is just the number of 1's in the `attestedBitmap`?

Comment (bruce-riley) [reply README.md:282]: That's right.

File: evm/src/interfaces/IEndpointIntegrator.sol [hunk 3]
```diff
@@ -48,26 +48,26 @@ interface IEndpointIntegrator is IMessageSequence {
     /// @param sequence The sequence number of the message.
     /// @param dstAddr The destination address of the message.
     /// @param payloadHash The keccak256 of payload from the integrator.
-    /// @return (uint128, uint128, bool) The enabled bitmap, the attested bitmap, if the message was executed.
+    /// @return (uint128, uint128, uint8, bool) The enabled bitmap, the attested bitmap, the number of attestations, if the message was executed.
```
Comment (panoel) [inline evm/src/interfaces/IEndpointIntegrator.sol:51]:
```suggestion
    /// @return (uint128, uint128, uint8, bool) The enabled bitmap, the attested bitmap, the number of attestations, and if the message was executed.
```

File: evm/src/interfaces/IEndpointIntegrator.sol [hunk 4]
```diff
@@ -48,26 +48,26 @@ interface IEndpointIntegrator is IMessageSequence {
...
+    ) external view returns (uint128, uint128, uint8, bool);
 
     /// @notice Retrieves the status of a message.
     /// @dev This version is expected to be called by the integrator on the destination chain.
     /// @param srcChain The Wormhole chain ID of the sender.
     /// @param srcAddr The universal address of the message.
     /// @param sequence The sequence number of the message.
     /// @param payloadHash The keccak256 of payload from the integrator.
-    /// @return (uint128, uint128, bool) The enabled bitmap, the attested bitmap, if the message was executed.
+    /// @return (uint128, uint128, uint8, bool) The enabled bitmap, the attested bitmap, the number of attestations, if the message was executed.
```
Comment (panoel) [inline evm/src/interfaces/IEndpointIntegrator.sol:66]:
```suggestion
    /// @return (uint128, uint128, uint8, bool) The enabled bitmap, the attested bitmap, the number of attestations, and if the message was executed.
```

**Review by panoel — APPROVED** [review]

**Review by panoel — APPROVED** [review]

**Review by evan-gray — APPROVED** [review]

**Comment by bruce-riley** [comment]
Opened issue #44 to document that other chains should add this return value as well.

**Pull request merged**